pkgs/os-specific/linux/intel-compute-runtime/default.nix

```diff
@@ -21,6 +21,8 @@ stdenv.mkDerivation rec {
     sha256 = "sha256-CWiWkv3CmHhXAk2M92voeQ06ximSOnT9hgIA4rIxWmM=";
   };
 
+  patches = [ ./fix_typo.patch ];
```

Comment:
Should use fetchpatch here.

Reply:
Why? I could link to https://patch-diff.githubusercontent.com/raw/intel/compute-runtime/pull/609.patch, but it's not upstreamed yet, and it might change.

Reply:
You can directly refer to the commit as https://github.com/intel/compute-runtime/commit/c54e968508aee6aa742e7b1514867a4b566fec7e.patch, which is immutable.